scripts/migrate.py

```diff
@@ -354,12 +361,12 @@ def migrate_trees(cursor, treenum=None, gedcom_path=None, on_save=None, dryrun=F
         logger.info('<<< migrated tree {}, path {}'
                     .format(row['GenTreeNumber'], file_name))
         count += 1
+
     return count
 
 
 if __name__ == '__main__':
     args = parse_args()
-    until = int(args.until)
```

Comment:
`until` variable is still referenced in the code below - looks like it will cause error